doc/reconstruction_models_list.rst

```diff
@@ -0,0 +1,222 @@
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Fadnavis 2019</a></td>
+    </tr>
+    <tr>
+        <td>SDT</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Descoteaux 2009</a></td>
+    </tr>
+    <tr>
+        <td><a href='#diffusion-spectrum-imaging'>DSI</a></td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td></td>
+        <td><a href=''>Wedeen 2008</a></td>
+    </tr>
+    <tr>
+        <td><a href='#dsi-with-deconvolution'>DSID</a></td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td></td>
+        <td><a href=''>Canales-Rodriguez 2010</a></td>
+    </tr>
+    <tr>
+        <td><a href='#generalized-q-sampling-imaging'>GQI - GQI2</a></td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='green'>Yes</td>
+        <td></td>
+        <td><a href=''>Yeh 2010</a></td>
+    </tr>
+    <tr>
+        <td><a href='#sparse-fascicle-model'>SFM</a></td>
+        <td class='green'>Yes</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Rokem 2015</a></td>
+    </tr>
+    <tr>
+        <td><a href='#q-ball-constant-solid-angle'>Q-Ball (OPDT)</a></td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Tuch 2004</a>, <a href=''>Tristan-Vega 2010</a></td>
+    </tr>
+    <tr>
+        <td><a href='#simple-harmonic-oscillator-based-reconstruction-and-estimation'>SHORE</a></td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td>
+            <a href=''>Merlet 2013</a>,
+            <a href=''>Özarslan 2009</a>,
+            <a href=''>Özarslan 2008</a></td>
+    </tr>
+    <tr>
+        <td><a href='#mean-apparent-propagator-map-mri'>MAP-MRI</a></td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Ozarslan 2013</a></td>
+    </tr>
+    <tr>
+        <td>MAPL</td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Fick 2016</a></td>
+    </tr>
+    <tr>
+        <td><a href='#constrained-spherical-deconvolution'>CSD</a></td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Tournier 2017</a></td>
+    </tr>
+    <tr>
+        <td>MS/MT CSD</td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href=''>Jeurissen 2009</a></td>
```

Comment:
Not 2009...

`B. Jeurissen, J.-D. Tournier, T. Dhollander, A. Connelly, and J. Sijbers. Multi-tissue constrained spherical deconvolution for improved analysis of multi-shell diffusion MRI data. NeuroImage, 103 (2014), pp. 411–426`